e2e-tests/helpers/boundary-http/accounts.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { nanoid } from 'nanoid';
+import { checkResponse } from './responseHelper.js';
+
+/**
+ * Creates a new password auth account
+ * @param {import('@playwright/test').APIRequestContext} request
+ * @param {string} authMethodId
+ * @param {string} username
+ * @param {string} password
+ * @returns {Promise<Serializable>}
+ */
+export async function createPasswordAccount(
+  request,
+  authMethodId,
+  username,
+  password,
```

Comment:
I've seen some methods follow the convention
```
export async function createHost(request, { hostCatalogId, name, address }) {
```
where the variables are in an object. Do we want to follow that format?